src/viam/resource/registry.py

```diff
@@ -151,6 +175,18 @@ def lookup_resource_creator(cls, subtype: "Subtype", model: "Model") -> "Resourc
             except KeyError:
                 raise ResourceNotFoundError(subtype.resource_type, subtype.resource_subtype)
 
+    @classmethod
+    def lookup_validator(cls, subtype: "Subtype", model: "Model") -> Union["Validator", None]:
```

Comment:
What do people think of returning a no-op instead of `None` here? Something like:

```python3
return cls._VALIDATORS.get(f"{subtype}/{model}", lambda x: return [])
```

Pros:
1. You don't have to check whether a validator exists

Cons:
1. You don't really *know* if a validator exists. But how important is it for that information to be accurate? Do you care if a validator exists? Or do you simply care to call it and have the correct output?

Reply:
That sounds good to me. I don't think we care if the validator exists; there's no special logic in that case except that we want to return an empty list of strings back to the parent.